feed/tests/test_comment_views.py

```diff
@@ -48,29 +56,27 @@ def test_create_new_comment_view_entrypoint_unauth(self, post, client):
         assert response.status_code == REDIRECT_URL_STATUS
         assert response.url == LOGIN_URL + '?next=' + new_comment_url(post.id)
 
-    def test_comment_creation_using_form(self, users, post, client):
+    def test_comment_creation_using_form(self, post, logged_in_client):
         # Testing that a comment creates successfully using the create new
         # comment form
         form_data = {'content': 'some coment content...'}
-        assert not Comment.comments.filter(content=form_data['content']).exists()
-        client.force_login(users[0])
-        client.post(new_comment_url(post.id), form_data)
-        assert Comment.comments.filter(content=form_data['content']).exists()
+        initial_comments_no = Comment.comments.count()
+        logged_in_client.post(new_comment_url(post.id), form_data)
```

Comment:
same as other PR
All asserts are done on the HTTP response, generated by the view
so you should receive the comment count from the returned response

Reply:
Done